src/nix/eval.cc

```diff
@@ -11,11 +11,13 @@
 
 using namespace nix;
 
+namespace nix::fs { using namespace std::filesystem; }
+
```

Comment:
nitpick(non-blocking): Could maybe even move this inside of the class declaration.

```suggestion
using fs = std::filesystem;

```

Reply:
Actually it can't be that, I'll say why below